src/person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Ahad', parent_permission: true)
```

Comment:
- Optional. 
I would recommend defaulting the name to unknown instead of your name since if the name is not provided, it will take your name as default and the user wouldn't know whether the name is unknown or provided.